schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+  id INTEGER,
```

Comment:
- [Optional] Consider making the id the primary key of the table and also for it not to be nullable and for the id to be autoincremented.